pkg/virt-operator/resource/generate/components/daemonsets.go

```diff
@@ -123,7 +123,7 @@ func NewHandlerDaemonSet(namespace, repository, imagePrefix, version, launcherVe
 				"node-labeller.sh",
 			},
 			SecurityContext: &corev1.SecurityContext{
-				Privileged: boolPtr(true),
+				Privileged: pointer.Bool(true),
```

Comment:
I do not think we should encourage using this one.
Either use the k8s `ptr` one or the in-house `pointer` one.